about-page.md

```diff
@@ -0,0 +1,47 @@
+
+Every group on karrot is completely independent and can have their own logo, processes and rules. Yet they are all displayed on the map so that people can find and connect to other groups in their location  and even globally.
+
+Have you already found a [group nearby you](https://karrot.world/)? Or maybe you want to [start one](https://karrot.world/#/group/create)? If you just want to try Karrot out, create an account and join the Playgroud group for testing.
+
+### Features
+
+[Improve the feature description at https://foodsaving.world/karrot
+and describe core features accompanied by icons or small screenshots]
+Here are some of Karrot's main features, in a nutshell.
+
+- Clear distribution of tasks and activities: Activities with a specific time and place, like food pickups in the case of foodsaving, can be entered in the software. Each activity has a number of slots which members of the group can sign up for – this way everyone knows who goes where and when, and feedback can be given later.
+
+- Share things: create an ad-like offer of whatever you want to share or give away. Offers are visible to everyone in the group and people can show their interest on the offer's chat.
+
+- Communication in the right context: write a message that everyone in the group will see or a private message to someone. Everything related to a single activity or place can be discussed in their respective chats.
+
+- Trust and group management: People in the group who receive enough trust-carrots from their peers will be able to edit and change settings in the group. When new people want to join your group they need to apply and answer questions your group can set.
+
+And more... [perhaps this part is collapsed and can be made visible by clicking on it?]
+
+- Maps and lists: You have a map of where the places and the people doing the activities are located. You have a nice list of upcoming activities and can sign up for them online.
+
+- Transparency: The history shows who did what and when. Everything from participating in an activity to creating and editing places is logged here.
+
+- Notifications: You get notified about empty activities, new messages and new people wanting to join your group. But you can also turn it all off if you wish more peace!
+
+- Multi-language: Karrot is fully translatable by the community and many languages are already added. To contribute more translations just visit the project on transifex.
+
+- Open source: Everything is transparent and possibly changeable. We encourage groups to host their own instances of the platform, if they have the ability to do so. Still, we also provide access to a hosted version on karrot.world.
+
+- Responsive team: You can easily talk to the people developing Karrot: Either on our forum, on Github or in real-time on our team chat (join the channel #foodsaving-worldwide or #karrot-dev to reach the right people immediately).
+
```

Comment:
I added this yesterday:
- Grassroots democracy:  If there is a conflict, the group can enter a voting system to resolve it in a communal way. There are no admins with super powers built into the software.

Reply:
Do you think this should be an item by itself in the "more", collapsible part or an item of the main features?

Reply:
Ok, so I chose to mix your suggestion with my other one after Nick's comment and put it under "Democratic and trust-based group management".
I feel that's a very long title to describe a set of features and the common idea behind them, but also more descriptive than "Grassroots democracy". So I don't know what to do of it.